java/src/main/java/org/rocksdb/TickerType.java

```diff
@@ -742,6 +742,16 @@
     COMPACT_WRITE_BYTES_PERIODIC((byte) -0x14),
     COMPACT_WRITE_BYTES_TTL((byte) -0x15),
 
+    /**
+     * DB error handler statistics
+     */
+    ERROR_HANDLER_BG_ERROR_COUNT((byte) -0x16),
+    ERROR_HANDLER_BG_IO_ERROR_COUNT((byte) -0x17),
+    ERROR_HANDLER_BG_RETRYABLE_IO_ERROR_COUNT((byte) -0x18),
+    ERROR_HANDLER_AUTORESUME_COUNT((byte) -0x19),
+    ERROR_HANDLER_AUTORESUME_RETRY_TOTAL_COUNT((byte) -0x1A),
```

Comment:
Would it make more sense for this to be a histogram stat, so we would get a distribution of how long the outages lasted?

Reply:
> Would it make more sense for this to be a histogram stat, so we would get a distribution of how long the outages lasted?

You mean for "ERROR_HANDLER_AUTORESUME_RETRY_TOTAL_COUNT"? This is just a general counter for the total retries being called since DB is opened.

 I used ERROR_HANDLER_AUTORESUME_RETRY_COUNT for histogram which shows how many times of retry will be called in one autoresume call. Did you mean this one?

Reply:
Yes. Sorry, I misread.